db/migrate/20210316202859_add_service_name_to_active_storage_blobs.active_storage.rb

```diff
@@ -0,0 +1,18 @@
+# This migration comes from active_storage (originally 20190112182829)
+class AddServiceNameToActiveStorageBlobs < ActiveRecord::Migration[6.0]
+  def up
+    unless column_exists?(:active_storage_blobs, :service_name)
+      add_column :active_storage_blobs, :service_name, :string
+
+      if configured_service = ActiveStorage::Blob.service.name
+        ActiveStorage::Blob.unscoped.update_all(service_name: configured_service)
+      end
```

Comment:
this seems like a weird migration:

1. create a brand new column
2. update all recrods with a new default
3. update column definition to disallow null columns

Can't 1-2 be combined into a single `add_column` with `default: configured_service` call? It seems that we're defining a service for all ./config/environments/ possible.

Or perhaps you can combine 2-3 with a call to [change_column_null](https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-change_column_null) providing a 4th argument of `configured_service`)

Reply:
Same here. These migrations were generated by the rails upgrade command so I never questioned the contents.